siriuspy/siriuspy/devices/bpm_fam.py

```diff
@@ -221,6 +221,46 @@ def set_attenuation(self, value=RFFEATT_MAX, timeout=TIMEOUT):
             _log.info(mstr)
         return okall
 
+    def set_tbt_mask(self, mask_begin, mask_end, timeout=TIMEOUT):
```

Comment:
I think we should have an input variable to decide whether we want to enable or disable the mask and also mask_begin and mask_end should have a default value of None, which means it would not set that property.

In this case, the suggestion bellow would not be valid and I think a refactoring of this method would be advised.